src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -387,14 +389,23 @@ private function getCartProductData(Cart $cart, Product $product, int $quantity)
      */
     private function addProductToCart(Cart $cart, Product $product, $combination, $quantity)
     {
+        /**
+         * Update product and customization in the cart, last argument skip quantity check
```

Comment:
It's a bit hard to understand when you are not "in the context" 😅 

May I suggest:
```
         * Here we update product and customization in the cart.
         *
         * The last argument "skip quantity check" is set to true because
         * 1) the quantity has already been checked,
         * 2) (main reason) when the cart checks the availability ; it substracts
         * its own quantity from available stock.
         *
         * This is because a product in a cart is not really out of the stock, because it is not checked out yet.
         *
         * Here we are editing an order, not a cart, so what has been ordered
         * has already been substracted from the stock.
```

Reply:
Yep much easier to read 😉